gnomad/utils/vcf.py

```diff
@@ -493,6 +493,31 @@ def make_combo_header_text(
     return header_text
 
 
+def create_label_groups(
+    pops: List[str],
+    sexes: List[str],
+    groups: List[str] = GROUPS,
+    pop_sex_groups: List[str] = ["adj"],
+) -> List[Dict[str, List[str]]]:
+    """
+    Generate a list of label group dictionaries needed to populate info dictionary.
+
+    Label dictionaries are passed as input to `make_info_dict`.
+
+    :param pops: List of population names.
+    :param sexes: List of sample sexes.
+    :param groups: List of data types (raw, adj). Default is ["raw", "adj"].
+    :param pop_sex_groups: List of only data types to populate with pops and sexes (raw, adj). Default is ["adj"].
+    :return: List of label group dictionaries.
+    """
+    return [
+        dict(group=groups),
```

Comment:
might be helpful to add why we need `groups` and `pop_sex_groups` (it isn't immediately obvious upon reading the code). I had a comment similar to the one below in the UKBB code
```suggestion
        # This is to capture raw frequency fields, which are
        # not stratified by sex or population (e.g., only AC_raw exists, not AC_XX_raw)
        dict(group=groups),
```